### PR TITLE
Mkdocs

### PR DESCRIPTION
**Pull request by nemchik**
Result should be https://linuxserver.github.io/docker-documentation/ via GitHub pages. Can be set to use custom domain later (to replace https://docs.linuxserver.io/ ).

Needs a bit of work for existing files with incorrect title formatting (which dictates navigation links) and table of contents on some pages (automatically generated by markdown h1-h6 when properly nested).

### CODE REVIEW

**Comment by nemchik** [comment]
Reverted snip stuff. Ready for review. We can do additional tinkering as a group once it's merged.

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: "/"
+    schedule:
+      interval: daily
```
Comment (Roxedus) [inline .github/dependabot.yml:6]: Do we need this frequent checks?

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+on: [push, pull_request]
+
+jobs:
+
+  build:
+    name: Build docs
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```
Comment (Roxedus) [inline .github/workflows/deploy.yml:11]: Could be specified as we have dependabot

File: .github/dependabot.yml (hunk 1 above)
Comment (nemchik) [reply .github/dependabot.yml:6]: Probably not. Would you think weekly?

File: .github/workflows/deploy.yml (hunk 2 above)
Comment (nemchik) [reply .github/workflows/deploy.yml:11]: A while ago i was under the impression that `@v3` meant "the latest 3.x.x" but I currently believe that is incorrect and it actually means "specifically 3.0.0". I'm file with this being more specific. As you said, dependabot will handle the updates.

Comment (Roxedus) [reply .github/workflows/deploy.yml:11]: v3 does mean that, but i think letting dependabot handle it is best

File: .github/dependabot.yml (hunk 1 above)
Comment (Roxedus) [reply .github/dependabot.yml:6]: yea, we dont really jump on dependabot PRs instantly anyway

Comment (nemchik) [reply .github/dependabot.yml]: Resolved in https://github.com/linuxserver/docker-documentation/pull/64/commits/998421310c107c7ff6c61c292fb3aad470585263

File: .github/workflows/deploy.yml (hunk 2 above)
Comment (nemchik) [reply .github/workflows/deploy.yml]: Resolved in https://github.com/linuxserver/docker-documentation/pull/64/commits/bfcdc2bb56fbf0a97cd04cb1c3d99aeb8f6e4936

**Pull request merged**